.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,9 @@
+{
+  "ExpandedNodes": [
+    "",
+    "\\PowerSlice",
+    "\\PowerSlice\\Properties"
+  ],
+  "SelectedNode": "\\PowerSlice\\Properties\\AssemblyInfo.cs",
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
I don't think this file should be added

Reply:
I can't put comments on the binaries, but I'm doubtful the .vs/slnx.sqlite should be included?

Reply:
I'm going to exclude the whole .vs folder. It contains status info for the specific Visual Studio user that is working on the project and should not be in repo. Will be rebuild when someone opens the solution locally